apps/activejobs/app/assets/javascripts/application.js.erb

```diff
@@ -218,6 +218,27 @@ function create_datatable(options){
                 data:               "cluster_title",
                 className:          "small",
                 "autoWidth":        true
+            },
+            {
+                data:               null,
+                className:          "small",
+                "autoWidth":        true,
+                "render":           function(data){
+                  console.log(data)
+                  if(data.username == JobStatusapp.username){
+                    return "<div>" +
+                              "<a class='btn btn-danger action-btn' data-method='delete'" +
+                              " data-confirm='Are you sure you want to delete " + data.jobname  + " - " + data.pbsid + "'" +
+                              " href='" + JobStatusapp.delete_job_path(data.pbsid, data.cluster) + "'" +
```

Comment:
The delete job path should be added to the JSON response instead of built here.
The way it can work is if you can delete the job, the delete button appears.

So there would be an extra attr in the json for each job, and most of the jobs would have that attr be empty.